WooCommerce/Classes/POS/Presentation/Item Search/POSPreSearchView.swift

```diff
@@ -0,0 +1,87 @@
+import SwiftUI
+import enum Yosemite.POSItem
+
+@available(iOS 17.0, *)
+struct POSPreSearchView: View {
```

Comment:
The empty recent search is saved after adding a popular item variation to the cart:

https://github.com/user-attachments/assets/286815f6-bc8e-4739-85c4-9ce7c068d3c2

Reply:
Aah, thank you! I'd seen an empty recent search in the history on one of my simulators, and thought it got there from early development of the search history feature. This makes more sense, thanks for the pointer.

Reply:
Strangely enough, I can't reproduce this. I've added a check to prevent it happening anyway, in 34fffc2, but it's a little odd as the steps are so simple.